src/runtime/virtcontainers/monitor.go

```diff
@@ -140,17 +140,14 @@ func (m *monitor) stop() {
 }
 
 func (m *monitor) watchAgent(ctx context.Context) {
-	err := m.sandbox.agent.check(ctx)
-	if err != nil {
+	if err := m.sandbox.agent.check(ctx); err != nil {
 		// TODO: define and export error types
 		m.notify(ctx, errors.Wrapf(err, "failed to ping agent"))
 	}
 }
 
-func (m *monitor) watchHypervisor(ctx context.Context) error {
+func (m *monitor) watchHypervisor(ctx context.Context) {
```

Comment:
why remove this method's return value error ?

Reply:
Thanks for catching that, I'll bring it back. The error in the caller is ignored, though.